Integrations/integration-Threat_Crowd.yml

```diff
@@ -116,13 +118,27 @@ script:
                 }
                 current[values[j]] = obj[k];
             });
+
+            if (dbotScore == 3) {
+                res.Malicious = {
+                   "Vendor": 'Threat Crowd',
+                   "Description": 'Votes field is -1'
```

Comment:
I think we should have a better explanation here

Reply:
Changed